packages/code-snippet/README.md

```diff
@@ -8,30 +9,31 @@ This package is build depending on:
 - and the [corresponding angular2 npm package](https://www.npmjs.com/package/angular2-highlight-js).
 - the [Marker npm package](https://www.npmjs.com/package/marked)
 
-*If you were to link this component locally, perform an `npm install` for the above npm packages in your app first.*
+## Usage
 
-### Highlight.js CDN's
-Add the following CDN's to your project:
+```typescript
+import 'CodeSnippetModule' from '@acpaas-ui/ngx-components/code-snippet'`;
+```
 
-- `<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.10.0/styles/atom-one-light.min.css">`
-- `<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.10.0/highlight.min.js"></script>`
+## Documentation
 
-You can also add additional languages if that's needed:
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
 
-- `<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.10.0/languages/typescript.min.js"></script>`
-- `<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.10.0/languages/scss.min.js"></script>`
-- ...
+### Code snippet list modules
 
-An overview of all CDN links can be found [here](https://cdnjs.com/libraries/highlight.js/)
+#### API
 
-## Installation
-```
-npm install @acpaas-ui/code-snippet --save
-```
+| Name         | Description | Documentations |
+| -----------  | ------ | -------------------------- |
+| `@Input() codeSnippet: string;` | `''` | Add your code snippet here. |
+| `@Input() processMarkdown: boolean;` | `false` | When having some markdown that contains code snippets. |
+| `@Input() scrollable: boolean;` | `true` | Boolean for when code snippet should not have a horizontal scrollbar when the code snippet is not wide enough. |
+
+#### Example
 
-Import the CodeSnippetModule in **app.module.ts**
+##### A single code snippet
 
-``` typescript
+```
```

Comment:
For consistency with the other examples, better to provide syntax typing.
`typescript`